github-archive/schema.sql

```diff
@@ -12,6 +11,7 @@ CREATE TABLE github (
     repo OBJECT AS (
       id LONG
     ),
+    payload_ft STRING INDEX using fulltext with (analyzer = 'english'),
```

Comment:
use a better analyzer, since the payload objects are json strings: https://crate.io/docs/stable/sql/analyzer.html#standard